xml/templates/message_templates/event_offline_receipt_html.tpl

```diff
@@ -213,7 +213,7 @@
         {/if}
        {/foreach}
        {if !empty($dataArray)}
-        {if isset($totalAmount) and isset($totalTaxAmount)}
+        {if $totalAmount and $totalTaxAmount}
```

Comment:
This would change the output for $0. This came up before didn't it? What was decided?

Reply:
@demeritcowboy yeah - I think we decided it was arguable enough whether it was better or worse it didn't matter